bin/readme.txt

```diff
@@ -0,0 +1,7 @@
+Go
+ - JSON serializatie van tijd
```

Comment:
Misschien beter om Engels te gebruiken?
Misschien ook niet handiger om deze inhoud op te nemen in `/README.md`?
Of deze bestand verplaatsen naar `go` directory?